universal-login-relayer/test/integration/core/services/MessageHandler.js

```diff
@@ -76,7 +77,7 @@ describe('INT: MessageHandler', async () => {
       it('should remove request from pending authorisations if addKey', async () => {
         const request = {walletContractAddress: walletContract.address, key: otherWallet.address, deviceInfo: defaultDeviceInfo};
         await authorisationStore.addRequest(request);
-        msg = {...addKeyMessage, from: walletContract.address, gasToken: mockToken.address, to: walletContract.address, nonce: await walletContract.lastNonce()};
+        msg = {...addKeyMessage, from: walletContract.address, gasToken: mockToken.address, to: walletContract.address, nonce: await walletContract.lastNonce(), gasData: 11408};
```

Comment:
Let's use `messageToSignedMessage` instead of `createSignedMessage`, so we don't need to provide `gasData`, which makes tests sensitive to changes in the data field and/or gas calculations.

#messageToSignedMessage